IdentityServer/v7/docs/content/diagnostics/otel/_index.md

```diff
@@ -18,3 +18,10 @@ especially in highly distributed systems.
 
 {{%children style="h4" /%}}
```

Comment:
I don't like the children just inserted here. They format in a confusing way. As a reader I wonder if those are meant to be headings with text below, or if there's a bug in the docs because it's blank. Also, the last one seems like a heading for the text below the children.